mod/00_utility/set/abstract/utility.rb

```diff
@@ -0,0 +1,8 @@
+
+def params_to_hash params
+  params.each_with_object({}) do |param, hash|
+    if (val = Env.params[param])
+      hash[param.to_sym] = val
+    end
+  end
```

Comment:
= `Env.params.select { |key, val| val && params.include?(key)}.symbolize_keys`

Maybe `fetch_params` or `select_params` describes better what that method does